simtools/configuration/commandline_parser.py

```diff
@@ -194,7 +194,7 @@ def initialize_db_config_arguments(self):
         _job_group.add_argument(
             "--db_simulation_model",
             help="name of simulation model database",
-            type=str,
+            type=self.strip_string,
```

Comment:
Can't you just do `type=str.strip` to achieve the same?

Reply:
Yes, exactly - this is simpler. Changed it (and changed also the corresponding unit test)

(I've done the function because I tried with with `type=str.strip()` and got an error. Clearly didn't think enough...)